index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>My Portfolio</title>
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+ integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+ crossorigin="anonymous" referrerpolicy="no-referrer" />
+<link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header>
+    <nav>
+      <a href="./index.html">Ashna Ali</a>
+      <img src="./Union.jpg" alt="menu" class="menu-icon">
+    </nav>
+  </header>
+    <main>
+ <section class="main-section">
```

Comment:
- [ ] As stated the project requirement and suggested by the previous reviewer,  the content of the **headline section** should be aligned with `flex-box`,This gives us a lot of options to align all our elements, plus, it's responsive.

![Capture](https://user-images.githubusercontent.com/93598189/212017333-0416a077-ef89-492c-8622-c277834fa8bf.PNG)

Reply:
how can i do it Can you please tell me?
@Abenezer-Tilahun

Reply:
@Abenezer-Tilahun  Do I need to add display: flex in the " * " container?  Kindly guide me then I will do it better also tell me how can I curve the page from bottom-left please guide me.

Reply:
Hello @Ashnaali3255 in-order to layout the headline section . just like this : 
```
.main-section {
display: flex;
}
```

Reply:
@Abenezer-Tilahun Thank you and can you tell me how can I fix it? Please guide me.[https://github.com/Ashnaali3255/Portfolio-2/pull/1#discussion_r1067081283](url)

Reply:
@Abenezer-Tilahun  Can we collaborate in the zoom, Please?

Reply:
Hello @Ashnaali3255 I see that you used `flexbox`  for the main container , I think it's OK but for future activities it better to layout the headline section "main section " using `flexbox`.

Reply:
@Abenezer-Tilahun  so now what do I have to do?